app/views/storage_locations/_source.html.erb

```diff
@@ -1,7 +1,7 @@
 <%
   storage_locations ||= @storage_locations
-  label ||= "From storage location"
-  error ||= "Which location are you moving inventory from?"
+  label ||= "Storage location"
+  error ||= "What location are you auditing?"
```

Comment:
This seems like it might affect another page - are we sure this was a flat out mistake rather than accidentally reusing logic in a place where we shouldn't be?

Reply:
@dorner I'm not sure I fully understand your question, so bear with me, please 🙏  This partial view is where the form labels are coming from for the new audit. Are you suggesting we make a new partial for the new view, instead of changing the existing partial that potentially affects another page?